### PR TITLE
Updated HCC voms cert

### PR DESCRIPTION
**Pull request by hzhu16**
Certificate's DN changed from ST=NE to ST=Nebraska

### CODE REVIEW

**Comment by brianhlin** [comment]
@edquist please review this, cut a release + build, and create a JIRA ticket

File: vomses [hunk 1]
```diff
@@ -18,6 +18,7 @@
 "GridUNESP" "voms.grid.unesp.br" "15000" "/C=BR/O=ANSP/OU=ANSPGrid CA/OU=Services/CN=voms.grid.unesp.br" "GridUNESP"
 "hcc" "hcc-voms.unl.edu" "15000" "/DC=org/DC=opensciencegrid/O=Open Science Grid/OU=Services/CN=hcc-voms.unl.edu" "hcc"
 "hcc" "hcc-voms2.unl.edu" "15000" "/DC=org/DC=incommon/C=US/ST=NE/L=Lincoln/O=University of Nebraska-Lincoln/OU=Holland Computing Center/CN=hcc-voms2.unl.edu" "hcc"
```
Comment (edquist) [inline vomses]:
```suggestion
```

Did you intend to leave in the old vomses entry?

```
[edquist@mystras ~/scratch/git/osg-vo-config]$ ./bin/vomses-crosscheck 
DNs don't match for hcc/hcc-voms2.unl.edu.lsc:
  vomses: /DC=org/DC=incommon/C=US/ST=NE/L=Lincoln/O=University of Nebraska-Lincoln/OU=Holland Computing Center/CN=hcc-voms2.unl.edu
     lsc: /DC=org/DC=incommon/C=US/ST=Nebraska/L=Lincoln/O=University of Nebraska-Lincoln/OU=Holland Computing Center/CN=hcc-voms2.unl.edu
```

**Comment by hzhu16** [comment]
Please remove the old vomses entry (with ST=NE) .
Thanks!

Huijun Zhu
Holland Computing center
huijun.zhu@unl.edu<mailto:huijun.zhu@unl.edu>



On Feb 11, 2020, at 2:07 PM, edquist <notifications@github.com<mailto:notifications@github.com>> wrote:


@edquist commented on this pull request.

________________________________

In vomses<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_opensciencegrid_osg-2Dvo-2Dconfig_pull_68-23discussion-5Fr377869858&d=DwMFaQ&c=Cu5g146wZdoqVuKpTNsYHeFX_rg6kWhlkLF8Eft-wwo&r=yd8AupC15a1kP8Tt1_Nf5Bnoer8osUd-1AnfXnqyGuE&m=rraKf-XaZLaMJjkR4X6RZOK8Bf9cauOtesuwOF_uxoE&s=zZcvMjGhc6GTfVvl1S2_YBMuzQ4rEucRxY64RyQ-H1I&e=>:

> @@ -18,6 +18,7 @@
 "GridUNESP" "voms.grid.unesp.br<http://voms.grid.unesp.br>" "15000" "/C=BR/O=ANSP/OU=ANSPGrid CA/OU=Services/CN=voms.grid.unesp.br<http://voms.grid.unesp.br>" "GridUNESP"
 "hcc" "hcc-voms.unl.edu<http://hcc-voms.unl.edu>" "15000" "/DC=org/DC=opensciencegrid/O=Open Science Grid/OU=Services/CN=hcc-voms.unl.edu<http://hcc-voms.unl.edu>" "hcc"
 "hcc" "hcc-voms2.unl.edu<http://hcc-voms2.unl.edu>" "15000" "/DC=org/DC=incommon/C=US/ST=NE/L=Lincoln/O=University of Nebraska-Lincoln/OU=Holland Computing Center/CN=hcc-voms2.unl.edu<http://hcc-voms2.unl.edu>" "hcc"


⬇️ Suggested change

-"hcc" "hcc-voms2.unl.edu<http://hcc-voms2.unl.edu>" "15000" "/DC=org/DC=incommon/C=US/ST=NE/L=Lincoln/O=University of Nebraska-Lincoln/OU=Holland Computing Center/CN=hcc-voms2.unl.edu<http://hcc-voms2.unl.edu>" "hcc"


Did you intend to leave in the old vomses entry?

[edquist@mystras ~/scratch/git/osg-vo-config]$ ./bin/vomses-crosscheck
DNs don't match for hcc/hcc-voms2.unl.edu.lsc:
  vomses: /DC=org/DC=incommon/C=US/ST=NE/L=Lincoln/O=University of Nebraska-Lincoln/OU=Holland Computing Center/CN=hcc-voms2.unl.edu<http://hcc-voms2.unl.edu>
     lsc: /DC=org/DC=incommon/C=US/ST=Nebraska/L=Lincoln/O=University of Nebraska-Lincoln/OU=Holland Computing Center/CN=hcc-voms2.unl.edu<http://hcc-voms2.unl.edu>

**Comment by edquist** [comment]
> Please remove the old vomses entry (with ST=NE) .

I don't have permission to update your branch, but if you click the "Commit suggestion" link here:
https://github.com/opensciencegrid/osg-vo-config/pull/68#discussion_r377869858

that should remove the old ST=NE line for you.

**Pull request merged**